src/pass.h

```diff
@@ -165,6 +166,15 @@ class Printer : public Pass {
   void run(PassRunner* runner, Module* module) override;
 };
 
+// Prints out a minified module
```

Comment:
Do we need this here? Printer is here so other places can use it, e.g. for debugging. But if this is only used by the user specifying it as a commandline pass, it doesn't need to be here.